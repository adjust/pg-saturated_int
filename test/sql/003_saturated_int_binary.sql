CREATE TABLE before (i saturated_int);
CREATE TABLE after (i saturated_int);

INSERT INTO before VALUES (1), (-1), (2147483649), (-2147483649);
SELECT * FROM before;

COPY before TO '/tmp/tst' WITH (FORMAT binary);
COPY after FROM '/tmp/tst' WITH (FORMAT binary);

SELECT * FROM after;
