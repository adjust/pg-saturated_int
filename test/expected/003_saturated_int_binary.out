CREATE TABLE before (i saturated_int);
CREATE TABLE after (i saturated_int);
INSERT INTO before VALUES (1), (-1), (2147483649), (-2147483649);
SELECT * FROM before;
      i      
-------------
 1
 -1
 2147483647
 -2147483648
(4 rows)

COPY before TO '/tmp/tst' WITH (FORMAT binary);
COPY after FROM '/tmp/tst' WITH (FORMAT binary);
SELECT * FROM after;
      i      
-------------
 1
 -1
 2147483647
 -2147483648
(4 rows)

